WcaOnRails/lib/custom_i18n_scanner.rb

```diff
@@ -19,6 +19,16 @@ def extract_model_name(key)
     when "devise" then "user"
     when "admin" then "person"
     when "oauth" then "doorkeeper/application"
+    when "contact"
+      # ContactsController uses WebsiteContact or DobCotact (which are much like extended models).
+      # We need to determine which one does the key refer to.
+      if key.include?("website")
+        "website_contact"
+      elsif key.include?("dob")
+        "dob_contact"
+      else
+        throw "Unrecognized contact model. Key: #{key}"
+      end
```

Comment:
It's just a nitpick, but I'd go with a `key.start_with?("contact.website")` just to be sure we don't hit a nested "dob"/"website" (which is obviously not the case right now, just thinking that we may add more contact models later)